test/units/module_utils/common/text/formatters/test_human_to_bytes.py

```diff
@@ -0,0 +1,81 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019, Andrew Klychkov @Andersson007 <aaklychkov@mail.ru>
+# Simplified BSD License (see licenses/simplified_bsd.txt or https://opensource.org/licenses/BSD-2-Clause)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+import pytest
+
+from ansible.module_utils.common.text.formatters import human_to_bytes
+
+
+@pytest.mark.parametrize(
+    'input_data,expected',
+    [
+        (0, 0),
+        (1024, 1024),
+        (u'1024B', 1024),
+        (u'0B', 0),
+        (u'1K', 1024),
+        (u'1KB', 1024),
+        (u'1MB', 1048576),
+        (u'1M', 1048576),
+        (u'1G', 1073741824),
+        (u'1GB', 1073741824),
+        (u'1T', 1099511627776),
+        (u'1TB', 1099511627776),
+        (u'1P', 1125899906842624),
+        (u'1PB', 1125899906842624),
```

Comment:
I feel like this could be a good place to use hypothesis...

Reply:
@webknjaz , hey :) could you explain your suggestion please?

Reply:
I found the info, thank you! need to sort it out